Tests/test_PDB_vectors.py

```diff
@@ -36,6 +36,16 @@ def test_legacy_imports(self):
             del vector_to_axis, Vector
 
 
+class VectorTests(unittest.TestCase):
+    """Tests for the Vector class."""
+
+    def test_division(self):
+        """Confirm division works."""
+        from Bio.PDB.Vector import Vector
+        v = Vector(2, 2, 2) / 2
+        self.assertEqual(repr(v), "<Vector 1.00, 1.00, 1.00>")
```

Comment:
I'm probably being too picky but this test wouldn't fail if the vector was somehow divided using floor division then cast to int. I'd use example numbers that would look even more strange if floor division were used (e.g. the classic `1 // 2 == 0`).

Reply:
I wondered that too - this would be clearer, good idea.